package-res/reportviewer/messages/messages_de.properties

```diff
@@ -22,3 +22,9 @@ SessionExpiredComment=Ihre Sitzung ist abgelaufen. Bitte melden Sie sich erneut
 
 ErrorParsingParamXmlMessage=Fehler beim Analysieren der Parameterinformation.
 FatalErrorTitle=Schwerwiegender Fehler
```

Comment:
Dont provide any translations unless you speak the language. Google translate is rather bad at that task, and the German here is nothing you'd want to see in a professional product ;)